ios/coredevice/coredevice.go

```diff
@@ -0,0 +1,20 @@
+package coredevice
+
+import "github.com/google/uuid"
+
+func BuildRequest(deviceId, feature string, input map[string]interface{}) map[string]interface{} {
```

Comment:
@dmissmann @diegoperini  why is this a one file module?  An explanation would be cool, I can add it to the docs